data/data/bootstrap/baremetal/files/etc/NetworkManager/dispatcher.d/30-local-dns-prepender

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
how is this file different from the mco PR https://github.com/openshift/machine-config-operator/pull/1396/files# ?? I don't see any such file in that PR.

Reply:
"30-resolv-prepender" - that one adds the DNS VIP to resolv.conf.  On the bootstrap host we just need localhost added, which is what this PR does.